mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loanaccount/LoanAccountFragment.java

```diff
@@ -199,20 +199,20 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
         inflatedisbusmentDate();
         inflateLoansProductSpinner();
 
-        disbursementon_date = tv_disbursementon_date.getText().toString();
-        submittion_date = tv_submittedon_date.getText().toString();
+        disbursementon_date = tvDisbursementOnDate.getText().toString();
+        submittion_date = tvSubmittedOnDate.getText().toString();
         submittion_date = DateHelper.getDateAsStringUsedForCollectionSheetPayload
                 (submittion_date).replace("-", " ");
         disbursementon_date = DateHelper.getDateAsStringUsedForCollectionSheetPayload
                 (disbursementon_date).replace("-", " ");
 
 
-        bt_loan_submit.setOnClickListener(new View.OnClickListener() {
+        btLoanSubmit.setOnClickListener(new View.OnClickListener() {
```

Comment:
We are using ButterKnife. Please remove this OnClickListener implementation and use Butter Knife ```@OnClick``` annotation.